### PR TITLE
Log metrics for CloudWatch Logs Insights

### PR DESCRIPTION
**Pull request by birme**
Addresses ticket #43

### CODE REVIEW

File: api/routes.js [hunk 1]
```diff
@@ -746,13 +747,18 @@ module.exports = (fastify, opt, next) => {
...
 
         reply.header("Content-Type", "application/xml; charset=utf-8");
         reply.code(200).send(vast_xml);
       }
     } catch (exc) {
-      logger.error(exc, { label: req.headers['host'], sessionId: session.sessionId });
+      if (session) {
+        logger.error(exc, { label: req.headers['host'], sessionId: session.sessionId });
+      } else {
+        logger.error(exc, { label: req.headers['host'] });
```
Comment (Nfrederiksen) [inline api/routes.js:760]: I believe that there are 4 more endpoints that use `session.sessionId` in logger.error(...) 
GET/sessions/:sessionId/tracking
GET/sessions/:sessionId/events
DELETE/sessions/:sessionId
GET/sessions/:sessionId

File: api/routes.js [hunk 2]
```diff
@@ -746,13 +747,18 @@ module.exports = (fastify, opt, next) => {
...
+          CloudWatchLog("ADS_RETURNED", req.headers['host'], { dur: session.adBreakDuration, session: session.sessionId });
         }
 
         reply.header("Content-Type", "application/xml; charset=utf-8");
         reply.code(200).send(vast_xml);
       }
     } catch (exc) {
-      logger.error(exc, { label: req.headers['host'], sessionId: session.sessionId });
+      if (session) {
+        logger.error(exc, { label: req.headers['host'], sessionId: session.sessionId });
```
Comment (Nfrederiksen) [inline api/routes.js:758]: not sure if `exc` by itself will let logger.error(...) display the message. 
`logger.error(exc.message, { label: req.headers['host'], sessionId: session.sessionId });`  might be needed.

Comment (birme) [reply api/routes.js:758]: Yes it does. I wanted to send in whatever was caught as it is not always an exception object is of a type containing the `.message`

File: api/routes.js (hunk 1 above)
Comment (birme) [reply api/routes.js:760]: Good point

**Pull request merged**